### DISCUSSION TITLE
Field Tram Lines

### DISCUSSION THREAD

**Discussion by Stu1986** [Courseplay/Courseplay_FS22#1896] [Q&A] [answered]
Hi All
I am getting used to PCs (coming from PS4), when using courseplay is the a trick to be able to get tram lines to allow for sprayers etc to not run over crops, I am using the field generator but unsure the best way to do it from them?

**Comment by Tensuko** [comment, marked as answer]
There is no trick and we won't implement this, sry.
It is alse game based that helper doesn't destroy crops.
If you want to use for example pro seed to do this, it won't be possible to pre calculate the path for the sprayer and stuff.

In my opinion, when I want to play that much realsitic, I should rather do this on my own :)
